pkg/artifact/jsonnetbundler/package.go

```diff
@@ -0,0 +1,126 @@
+package jsonnetbundler
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+
+	"github.com/neticdk/go-common/pkg/artifact"
+	"github.com/pkg/errors"
+
+	"github.com/jsonnet-bundler/jsonnet-bundler/pkg"
+	"github.com/jsonnet-bundler/jsonnet-bundler/pkg/jsonnetfile"
+	specv1 "github.com/jsonnet-bundler/jsonnet-bundler/spec/v1"
+	"github.com/jsonnet-bundler/jsonnet-bundler/spec/v1/deps"
+)
+
+const filePermissionNewFile = 0o640
+
+type PackageOptions struct {
+	JsonnetHome string
+}
+
+// Install installs a package and its dependencies in the vendor directory.
+// It mimics https://github.com/jsonnet-bundler/jsonnet-bundler/blob/master/cmd/jb/install.go
+func Install(_ context.Context, a *artifact.Artifact, opts *PackageOptions) (*artifact.PullResult, error) {
```

Comment:
The `context.Context` parameter is there to adhere to a certain interface? Otherwise I would delete it as long as it is unused.

Reply:
Removed